com.unity.render-pipelines.high-definition/Editor/Material/DiffusionProfile/DiffusionProfileMaterialUI.cs

```diff
@@ -42,6 +42,13 @@ public static void OnGUI(MaterialProperty diffusionProfileAsset, MaterialPropert
                 // encode back GUID and it's hash
                 diffusionProfileAsset.vectorValue = newGuid;
                 diffusionProfileHash.floatValue = hash;
+
+                // Update external reference.
+                foreach (var target in materialEditor.targets)
+                {
+                    MaterialExternalReferences matExternalRefs = MaterialExternalReferences.GetMaterialExternalReferences(target as Material);
```

Comment:
guess the call of GetMaterialExternalReferences should be outside the llop? to avoid to call loaddatabase each time?

Reply:
nevermind :)